src/librustc_mir/dataflow/at_location.rs

```diff
@@ -60,19 +61,24 @@ pub trait FlowsAtLocation {
 /// (e.g., via `reconstruct_statement_effect` and
 /// `reconstruct_terminator_effect`; don't forget to call
 /// `apply_local_effect`).
-pub struct FlowAtLocation<'tcx, BD>
+#[derive(Clone)]
+pub struct FlowAtLocation<'tcx, BD, DR>
```

Comment:
```suggestion
pub struct FlowAtLocation<'tcx, BD, DR = DataflowResults<'tcx, BD>>
```

Instead of adding a type alias, you could just default `DR` and leave all current uses unchanged.

Reply:
ah, I forgot we can use this in rustc :)